home.md

```diff
@@ -237,9 +237,9 @@ span.blue {
       <p>A short and high level overview of the Platform</p>
     </div>
     <div class="col-xs-12 col-sm-6">
-      <a href="https://www.pluralsight.com/courses/microservices-nservicebus6-scaling-applications">
-        <img src="/home/pluralsight_v1.svg" width="36" height="36">
-        <h3>Pluralsight: Scaling Applications with Microservices and NServiceBus 6</h3>
+      <a href="https://discuss.particular.net">
+        <img src="/content/images/discussiongroup_v1.svg" width="36" height="36">
+        <h3>Discussion group</h3>
```

Comment:
Why is this change in this PR?

Reply:
Because I wanted to keep a link to the discussion group which required to drop some of the existing links. Given the pluralsight course is fairly old by now and uses an outdated NSB version, I went with this option